target/linux/airoha/files/drivers/pinctrl/pinctrl-airoha.c

```diff
@@ -1604,8 +1854,8 @@ static u32 airoha_pinctrl_rmw(struct airoha_pinctrl *pinctrl,
 	return val;
 }
 
-#define airoha_pinctrl_mux_set(pinctrl, base, offset, val)			\
-	airoha_pinctrl_rmw((pinctrl), (base) + (offset), 0, (val));
+#define airoha_pinctrl_mux_set(pinctrl, base, offset, mask, val)			\
```

Comment:
@merbanan we do not need this change anymore

Reply:
Ack.